### ISSUE TITLE
Wrong version comparator?

### ISSUE THREAD

**Issue by Riggs333** [4finance/uptodate-gradle-plugin#25]
With the following dependency

```groovy
dependencies {
    compile 'org.webjars:angularjs:1.3.14'
}
```
running the uptodate task tells me

    New versions available:
    'org.webjars:angularjs:1.3.4-1'

The version 1.3.4-1 is older than the current version.
I expect the plugin to find the current version **org.webjars:angularjs:1.3.15**

**Comment by cbuleandra** [comment]
We'll look into it and try to fix it ASAP.

**Comment by cbuleandra** [comment]
Should be fixed in release 1.3.1.
